test/unit/spec/OAuth2Util_spec.js

```diff
@@ -58,6 +61,42 @@ describe('util/OAuth2Util', function() {
       }).catch(done.fail);
     });
 
+    it.each([
+      'access_denied',
+      'jit_failure_missing_fields',
+      'jit_failure_invalid_login_format',
+      'jit_failure_values_not_match_pattern',
+      'jit_failure_values_too_long',
+      'jit_failure_invalid_locale',
+    ])('shows SDK errors of certain types', function(errCode, done) {
+      const errorMessage = 'Auth SDK error';
+      const authException = new AuthSdkError(errorMessage);
+      authException.errorCode = errCode;
+
+      spyOn(authClient.token, 'getWithPopup').and.callFake(function() {
```

Comment:
nit: Not a blocker, but we should try to use [Jest Objects](https://jestjs.io/docs/jest-object) instead of Jasmine. The difference is subtle, but the method signatures are different:

```suggestion
      jest.spyOn(authClient.token, 'getWithPopup').mockImplementation(() => {
```